wcivf/apps/elections/views/mixins.py

```diff
@@ -177,11 +177,21 @@ def get_advance_voting_station_info(self, polling_station: Optional[dict]):
         advance_voting_station["open_in_future"] = open_in_future
         return advance_voting_station
 
-    def is_before_registration_deadline(self, post_elections):
-        if not post_elections:
+    def get_global_registration_card(self, post_elections):
+        # City of London local elections have different
+        # registration rules to every other election
+        non_city_of_london_ballots = [
+            ballot
+            for ballot in post_elections
+            if not ballot.election.is_city_of_london_local_election
+            and not ballot.cancelled
```

Comment:
More info in https://github.com/DemocracyClub/WhoCanIVoteFor/commit/cb3e726bab9b9198e39b4a3189cb890aec4029ed
I suspect some variation of this oversight exists in all sorts of other places in this codebase
I've fixed it here, but I haven't gone digging